kusto-language-server/package.json

```diff
@@ -66,6 +66,10 @@
             {
                 "command": "kuskus.loadSymbols",
                 "title": "[Kuskus] Load Symbols from Cluster"
+            },
+            {
+                "command": "kuskus.loadTable",
```

Comment:
Does this change fit with this PR? I don't recall seeing a `loadTable` command elsewhere.